Control-User-Interfaces/example-notebook/ExampleNotebook.rst

```diff
@@ -0,0 +1,225 @@
+
+.. note::
+
+	This page is automatically generated by exporting `this example notebook <https://github.com/lsst-ts/ts_notebooks/blob/develop/examples/WritingSALScript.ipynb>`_.
```

Comment:
This doesn't appear to be rendering correctly... none of the code blocks are showing..